back-end/routes/stitchRoutes.js

```diff
@@ -56,22 +56,27 @@ router.get('/title/:stitchId', async (req, res) => {
 });
 
 router.post('/create', async (req, res) => {
-    const { title, username, mood, dance, mix, explore } = req.body
+    const { title, username, mood, dance, mix, explore, imageUrl } = req.body
 
     const user = await prisma.user.findUnique({
         where: {
             username: username
         }
     })
 
+    const totalPreferences = mood + dance + mix;
+    const moodWeight = mood / totalPreferences;
+    const danceWeight = dance / totalPreferences;
+    const mixWeight = 1 - (moodWeight + danceWeight);
```

Comment:
Any reason why we're just not doing `const mixWeight  =  mix / totalPreferences;` ?

Reply:
> Any reason why we're just not doing `const mixWeight = mix / totalPreferences;` ?

I was doing that to make sure they add up to 1.

Reply:
Yeah that's okay